backend/drm/drm.c

```diff
@@ -598,6 +598,8 @@ static bool wlr_drm_output_set_cursor(struct wlr_output *_output,
 		wlr_matrix_texture(plane->matrix, plane->width, plane->height,
 			output->output.transform ^ WL_OUTPUT_TRANSFORM_FLIPPED_180);
 
+		// TODO the image needs to be rotated depending on the output rotation
```

Comment:
I think what could be happening is that if I simply multiply by a rotation matrix, it will rotate around the origin when what I really want is rotation around the center but I'm still not sure what's going on.